index.html

```diff
@@ -366,6 +375,8 @@ <h4>Global Attributes</h4>
             <li><a><code>scriptlevel</code></a></li>
             <li><a><code>style</code></a></li>
             <li><a><code>tabindex</code></a></li>
+            <li><a><code>intent</code></a></li>
```

Comment:
I wonder if these should not be handled like the handler attributes and just say "any of the reserved for future use attributes" and link to a section for that?